package.json

```diff
@@ -34,20 +34,24 @@
     "sass-loader": "^7.1.0",
     "style-loader": "^1.0.0",
     "stylelint-config-prettier": "^5.2.0",
+    "svg-react-loader": "^0.4.6",
     "webpack": "^4.39.1",
     "webpack-cli": "^3.3.6",
     "webpack-dev-server": "^3.7.2"
   },
   "dependencies": {
+    "@svgr/webpack": "^4.3.2",
     "axios": "^0.19.0",
+    "babel-plugin-named-asset-import": "^0.3.3",
+    "history": "^4.9.0",
```

Comment:
`@svgr/webpack` and `babel-plugin-named-asset-import` should be in devDependencies